### ISSUE TITLE
qt-gui: responsiveness

### ISSUE THREAD

**Issue by markus2330** [ElektraInitiative/libelektra#164]
Starting gui on a Intel(R) Xeon(R) CPU E3-1231 v3 @ 3.40GHz takes:

    23,50s user 0,77s system 93% cpu 25,948 total

creating a mountpoint takes about the same. Something weird is going on there.

kdb ls alone takes:

```
$ time kdb ls system | wc -l
1492
kdb ls system  0,06s user 0,00s system 96% cpu 0,070 total
wc -l  0,00s user 0,00s system 0% cpu 0,070 total
```

**Comment by 0003088** [comment]
Problem is fixed in 5416cf7.

**Comment by markus2330** [comment]
Improved to 0.5 sec constant time.

Tested without any keys and with 328 mountpoints holding 7778 keys.

Very good done!
